### ISSUE TITLE
renderJpg() color is  bluer than expected

### ISSUE THREAD

**Issue by jdenisgiguere** [locationtech/geotrellis#3203]
## Current situation

With I use `rendegJpg()` on a Multiband Tille, the color is not as expected. It is bluer than expected.

`renderPng()` return an image of the expected color.

See `renderJpg()` result
![renderJpg](https://user-images.githubusercontent.com/973241/76792921-7132ca80-679a-11ea-917c-30e1b10d6c94.jpg)

and `renderPng()` result
![renderPng](https://user-images.githubusercontent.com/973241/76792882-5a8c7380-679a-11ea-9068-8d7b936724bd.png)
 

## Expected situation

`repnderPng()` and `renderJpg()` should return similar image.

**Comment by pomadchin** [comment]
Hi! Thanks for reporting, it is a duplicate of #3151; could you upload an original small image that you use to generate pngs / jpgs so we can test against it?

**Comment by jdenisgiguere** [comment]
Thanks for the super quick follow up!

See: 
[tiled_U-19OrthoSectorTile1_2012x2012.zip](https://github.com/locationtech/geotrellis/files/4340036/tiled_U-19OrthoSectorTile1_2012x2012.zip)

**Comment by pomadchin** [comment]
@jdenisgiguere thanks!

**Comment by esmeetu** [comment]
Same here.

Problem here might be `ImageIO` recieve a `ARGB` while geotrellis give us a `RGBA`.
And i found a workaround:
```scala
  def renderJpg(img: MultibandTile) = {
    // get a rgba tile
    val coloredImg = img.color()

    // begin translate from rgba to argb
    val (cols, rows) = (img.cols, img.rows)
    val bi = new BufferedImage(cols, rows, BufferedImage.TYPE_INT_ARGB)
    cfor(0)(_ < cols, _ + 1) { x =>
      cfor(0)(_ < rows, _ + 1) { y =>
        val v = coloredImg.get(x, y)
        val r = (v >> 24) & 0xFF
        val g = (v >> 16) & 0xFF
        val b = (v >> 8) & 0xFF
        val a = v & 0xFF
        bi.setRGB(x, y, a << 24 | r << 16 | g << 8 | b)
      }
    }
    val result = ImageIO.write(bi, "png", new File("img.png"))
    print(result)
  }
```
here i use `BufferedImage.TYPE_INT_ARGB` to keep alpha band. However when i write to `jpg` format, it will output a wired `cmyk` format and black image. Using `png` format works well and saves the alpha info. 

In the end, i think it had better to make a change in the method `toBufferedImage` of file `ColorMethods`. In case we know that tile is `RGBA` format, we should make a trans in that method. Like this:
```scala
  def toBufferedImage: BufferedImage = {
    val bi = new BufferedImage(self.cols, self.rows, BufferedImage.TYPE_INT_RGB)
    cfor(0)(_ < self.cols, _ + 1) { x =>
      cfor(0)(_ < self.rows, _ + 1) { y =>
        val v = self.get(x, y)
        val r = (v >> 24) & 0xFF
        val g = (v >> 16) & 0xFF
        val b = (v >> 8) & 0xFF
        val a = v & 0xFF
        bi.setRGB(x, y, a << 24 | r << 16 | g << 8 | b)
      }
    }
    bi
  }
```

**Comment by pomadchin** [comment]
Hi @esmeetu that was exactly that! Should be fixed in terms of #3278 now.

cc @jdenisgiguere
